l10n_it_website_portal_fatturapa_sale/views/payment_templates.xml

```diff
@@ -9,11 +9,23 @@
               name="Add e-invoice subjected in Payment Tokens list"
               inherit_id="payment.payment_tokens_list">
         <xpath expr="//div[hasclass('card')]" position="after">
-            <div class="form-check">
+            <div class="form-check" t-att-style="'display:none;' if (sale_order and sale_order.company_id.portal_mandatory_e_invoicing) else ''">
```

Comment:
Se ho ben capito, questo dovrebbe far sì che abilitando `portal_mandatory..` non viene visualizzato il check `I request to receive..` ma 'sotto-sotto' il check è `True`.
Se è così, non mi pare funzionare (ho provato nel runbot della PR):
![Peek 2019-12-18 18-01](https://user-images.githubusercontent.com/32064796/71107115-cf163d00-21c0-11ea-8f40-061eaf6e0a0b.gif)

Reply:
Attenzione che questo modulo opera sugli ordini "portal", quindi i preventivi online, e non sugli ordini dell'e-commerce

Reply:
Giusto, ho provato su un ordine portale ed è tutto ok, grazie!